math/src/App.css

```diff
@@ -1,39 +1,63 @@
-.App {
-  text-align: center;
+.calculator-container {
```

Comment:
- [ ] Kindly remove the default files created by CRA.

<img src='https://user-images.githubusercontent.com/33323077/130680035-ab51807f-e3b5-4db5-80e6-c3ceba7250fd.png' height='250px'>